content/verification/veteran-confirmation/release-notes/2024-05-14.md

```diff
@@ -0,0 +1,24 @@
+Veteran Confirmation
+/status
+
+Post requests - Special characters and spaces are now accepted in Demographic fields as follows:
```

Comment:
Is this saying that for post requests, special characters and space are now accepted...? As in, this set of criteria only applies to post requests?

Reply:
Yes but there is only one kind of request. The single post request to confirm /status